life_expectancy/main.py

```diff
@@ -1,42 +1,57 @@
+import os
 import argparse
-from pathlib import Path
-from life_expectancy.data_loading import load_data
-from life_expectancy.data_loading import save_data
-from life_expectancy.data_cleaning import clean_data
+import pandas as pd
+import life_expectancy.data_loading as dl
+import life_expectancy.data_cleaning as dc
+from life_expectancy.region import Region
 
-def main(country: str) -> None:
+DEFAULT_INPUT_FILE = "life_expectancy/data/eu_life_expectancy_raw.tsv"
+DEFAULT_OUTPUT_FILE = "life_expectancy/output/pt_life_expectancy.csv"
+
+def get_file_extension(input_file: str) -> str:
+    """
+    Returns the extension of the file in the given path.
+    """
+    extension = os.path.splitext(input_file)
+    return extension[1]
```

Comment:
If you use `pathlib.Path` instead of strings, you can replace this function with `path.suffix`.